kodilla-testing/src/test/java/com/kodilla/testing/library/BookDirectoryTestSuite.java

```diff
@@ -81,4 +81,58 @@ public void testListBooksWithConditionFragmentShorterThan3() {
         Assert.assertEquals(0, theListOfBooks10.size());
         verify(libraryDatabaseMock, times(0)).listBooksWithCondition(anyString());
     }
+
+    @Test
+    public void testListBooksInHandsOfNoneBooks(){
+        //Given
+        LibraryDatabase libraryDatabaseMock = mock(LibraryDatabase.class);
+        BookLibrary bookLibrary = new BookLibrary(libraryDatabaseMock);
+        LibraryUser libraryUser = new LibraryUser("Jan", "Kowalski", "90051773287");
+        List<Book> expectedBooks = new ArrayList<>();
+
+        when(libraryDatabaseMock.listBooksInHandsOf(libraryUser)).thenReturn(expectedBooks);
+
+        //When
+        List<Book> result = bookLibrary.listBooksInHandsOf(libraryUser);
+
+        //Then
+        Assert.assertEquals(expectedBooks, result);
+    }
+
+    @Test
+    public void testListBooksInHandsOfOneBook(){
+        //Given
+        LibraryDatabase libraryDatabaseMock = mock(LibraryDatabase.class);
+        BookLibrary bookLibrary = new BookLibrary(libraryDatabaseMock);
+        LibraryUser libraryUser = new LibraryUser("Jan", "Kowalski", "90051773287");
+        List<Book> expectedBooks = new ArrayList<>();
+        expectedBooks.add(new Book("Title", "Author", 1999));
+
+        when(libraryDatabaseMock.listBooksInHandsOf(libraryUser)).thenReturn(expectedBooks);
+
+        //When
+        List<Book> result = bookLibrary.listBooksInHandsOf(libraryUser);
+
+        //Then
+        Assert.assertEquals(expectedBooks, result);
+        Assert.assertEquals(1, result.size());
```

Comment:
Czy ta asercja jest potrzebna? :) A jeśli już ją bardzo chcesz, to powinna być przed tą drugą asercją: zazwyczaj wpierw sprawdzamy, czy liczba elementów się zgadza, a potem dopiero patrzymy na te elementy

Reply:
Wstawiłem tę asercję, bo test ma być dla 5 książek, stąd chcę też sprawdzić, czy jest ich na pewno 5.

Reply:
Tutaj akurat dla 1 książki :)